src/Microsoft.VisualStudio.ProjectSystem.Managed/Build/BuildUtilities.cs

```diff
@@ -141,7 +141,7 @@ public static void RemovePropertyValue(ProjectRootElement project, string evalua
             bool valueFound = false;
             foreach (string value in GetPropertyValues(evaluatedPropertyValue, delimiter))
             {
-                if (!string.Equals(value, valueToRemove, StringComparisons.PropertyValues))
```

Comment:
This method looks like a generic helper method for any property. Are you sure its only called in places where spaces aren't important? Or are spaces not supported at all by MSBuild in this scenario?

Is it worth adding a note to the XML comments that says it is space-agnostic?

Reply:
This will break configurations `<Configuration>` with spaces in their names "Release (Production)".

I think any fix around this belongs in CPS - it already handles the mapping between "Any CPU" and "AnyCPU", see ProjectConfigProviderBase.ConvertVsConfigPlatformToProjectConfigPlatform usage. It should have already fixed this, if not, then that's the bug.